docx/oxml/simpletypes.py

```diff
@@ -298,6 +312,111 @@ class ST_Merge(XsdStringEnumeration):
 
     _members = (CONTINUE, RESTART)
 
+class ST_MultiLevelType(XsdStringEnumeration):
+    """
+    Valid values for <w:multiLevelType val=""> attribute
+    """
+    SINGLE = 'singleLevel'
+    MULTI = 'multilevel'
```

Comment:
I think the capitalization was correct viz. "multiLevel" for open standards

http://officeopenxml.com/WPnumberingAbstractNum.php

MS specifies what you have changed here:
https://docs.microsoft.com/en-us/dotnet/api/documentformat.openxml.wordprocessing.multilevelvalues?view=openxml-2.8.1

wondering if this will be tricky for compatibility across UAs